apps/web/src/app/(app)/create/_components/create-project-form.tsx

```
@@ -0,0 +1,235 @@
+          file.type === "text/plain";
+
+        if (!isValid) {
+          toast.error("Please upload only images, PDFs, or text files.");
+        }
+
+        return isValid;
+      });
+
+      return toast.info("Launching soon...");
+
+      if (allowedFiles.length === 0) return;
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)